app/views/discovered_hosts/_discovered_hosts_list.html.erb

```diff
@@ -35,7 +35,7 @@
       <% if SETTINGS[:organizations_enabled] -%>
         <td class="hidden-tablet hidden-xs"><%= host.organization.try(:name) %></td>
       <% end -%>
-      <td class="hidden-tablet hidden-xs"><%= host.subnet %></td>
+      <td class="hidden-tablet hidden-xs"><%= host.primary_interface.subnet %></td>
```

Comment:
It looks like V2 API / CLI does work well. No changes required there, this is the only place it looks like.